client/blocks/get-apps/index.tsx

```diff
@@ -0,0 +1,42 @@
+import config from '@automattic/calypso-config';
+import { useTranslate } from 'i18n-calypso';
+import NavigationHeader from 'calypso/components/navigation-header';
+import BodySectionCssClass from 'calypso/layout/body-section-css-class';
+import { createWordPressDesktopConfig, createWordPressStudioConfig } from './apps-config';
+import DesktopDownloadCard from './desktop-download-card';
+import MobileDownloadCard from './mobile-download-card';
+
+import './style.scss';
+
+export const GetApps = () => {
+	const translate = useTranslate();
+	const envId = config( 'env_id' );
+	const isDesktopEnv = typeof envId === 'string' && envId.startsWith( 'desktop' );
+
+	const desktopApps = [
+		createWordPressStudioConfig( translate ),
+		createWordPressDesktopConfig( translate ),
+	];
+
+	return (
+		<>
+			<NavigationHeader title={ translate( 'Apps' ) } />
+			<div className="get-apps__wrapper">
+				<BodySectionCssClass bodyClass={ [ 'get-apps__body' ] } />
```

Comment:
```suggestion
```
If we remove the styling for `body.get-apps__body` in `client/blocks/get-apps/style.scss`, then we can remove this component, too.